subsys/dfu/fmfu_fdev/Kconfig

```diff
@@ -0,0 +1,35 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+menuconfig FMFU_FDEV
+	bool "Full Modem Firmware Upgrade from flash device"
+	depends on CDDL_GEN
+
+
+if (FMFU_FDEV)
+
+config MBEDTLS_HEAP_SIZE
+	int
+	default 4096
```

Comment:
@tejlmand Won't this create ambiguity? I. e. two unconditional defaults for mbedTLS, shouldn't it rather be `default 4096 if FMFU_FDEV`? Or is the top-level `if` enough for Kconfig to handle that?

Reply:
well, the `if (FMFU_FDEV)` does help here, but because this file gets sourced before:
https://github.com/nrfconnect/sdk-zephyr/blob/master/modules/Kconfig.mbedtls#L123
it's default wins.

If it was sourced after Kconfig.mbedtls, then the default would be 512, and `f (FMFU_FDEV)` or a `default 4096 if FMFU_FDEV` wouldn't change that.